### PR TITLE
CLDR-11645 A Add aliases for identical units

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-11645
- [x] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by macchiati** [comment]
Peter was pulled away for a few days. Can one of you do this one?

**Comment by btangmu** [comment]
Build failures starting here?
```
    testExtraPaths
Units with grammar info: 73
Inflection Paths
4	hi
2	fr
 {
      Error: (TestCLDRFile.java:361) +en, -[root]	//ldml/units/unitLength[@type="long"]/unit[@type="energy-foodcalorie"]/displayName
...
```

**Comment by macchiati** [comment]
Thanks, missed that.

Mark

**Comment by macchiati** [comment]
Hi, can one of you now review, since the tests now pass?

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRFile.java [hunk 1]
```diff
@@ -299,8 +299,16 @@ public void testExtraPaths() {
                         || path.contains("[@alt=\"formal\"]")
                         || (path.contains("dayPeriod[@type=")
                             && (path.endsWith("1\"]") || path.endsWith("\"am\"]") || path.endsWith("\"pm\"]") || path.endsWith("\"midnight\"]"))) // morning1, afternoon1, ...
-                        || (path.startsWith("//ldml/characters/exemplarCharacters[@type=\"index\"]")
-                            && localeInfo.locale.equals("root"))
+                        || (localeInfo.locale.equals("root") &&
+                            (path.startsWith("//ldml/characters/exemplarCharacters[@type=\"index\"]")
```
Comment (btangmu) [inline tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRFile.java:303]: This brings the function up to 246 lines -- ideally it should be split up with a lot of subroutines; I'm trying to get in the habit of considering this when inserting code into functions that are already large

**Review by btangmu — APPROVED** [review]

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRFile.java (hunk 1 above)
Comment (macchiati) [reply tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRFile.java:303]: Yes, that is an overly-long test function. I agree it should be done, but want to focus now on the data changes for the freeze.

**Pull request merged**